a8c-files.php

```diff
@@ -490,24 +504,55 @@ function delete_file( $file_name ) {
 	private function purge_file_cache( $url, $method ) {
 		global $file_cache_servers;
 
+		$parsed = parse_url( $url );
+		if ( empty( $parsed['host'] ) ) {
+			return $requests;
+		}
+
+		$uri = '/';
```

Comment:
Minor: I don't think there's any major concerns with with this default since this is a `private` function, but we could end up sending a purge for `example.com/` if a non-file URL is passed through. Probably not worth changing but figured I'd flag it just in case.

Reply:
It wouldn't cause any issues if we did, so I think it's okay.